client/src/pages/DetailsPage/item-details.jsx

```diff
@@ -1,3 +1,55 @@
-export const ItemDetails = () => {
-	return <></>;
+import styles from './item-details.module.scss';
+import { GoBackButton } from "../../components/GoBackButton/GoBackButton";
+import React, { useState } from 'react';
+
+export const ItemDetails = (props) => {
+	const [quantity, setQuantity] = useState(0);
+
+	const incrementQuantity = () => {
+		setQuantity(previousQuantity => previousQuantity + 1);
+	};
+
+	const decrementQuantity = () => {
+		if (quantity > 0) {
+			setQuantity(previousQuantity => Math.max(previousQuantity - 1, 0));
+		}
```

Comment:
Das `IF` Statement ist nicht mehr nötig, da du mit `Math.max()` bereits sicherstellst, dass der Wert immer größer gleich 0 ist.
```suggestion
			setQuantity(previousQuantity => Math.max(previousQuantity - 1, 0));
```

Reply:
done!